### PR TITLE
Move examples directory out of source directory #221

### PR DESCRIPTION
**Pull request by shashvatshahi1998**
#221

### CODE REVIEW

File: examples/layers_weightnormalization.ipynb [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+        "colab_type": "text",
+        "id": "MfBg1C5NB3X0"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "# TensorFlow Addons Layers: WeightNormalization\n",
+        "\n",
+        "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/addons/blob/master/tensorflow_addons/examples/layers_weightnormalization.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
```
Comment (facaiy) [inline examples/layers_weightnormalization.ipynb]: Hi, don't forget to modify those links: remove `tensorflow_addons`

File: examples/layers_weightnormalization.ipynb [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://github.com/tensorflow/addons/blob/master/tensorflow_addons/examples/layers_weightnormalizations.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
```
Comment (facaiy) [inline examples/layers_weightnormalization.ipynb]: ditto

File: examples/optimizers_lazyadam.ipynb [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+        "colab_type": "text",
+        "id": "MfBg1C5NB3X0"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "# TensorFlow Addons Optimizers: LazyAdam\n",
+        "\n",
+        "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/addons/blob/master/tensorflow_addons/examples/optimizers_lazyadam.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
```
Comment (facaiy) [inline examples/optimizers_lazyadam.ipynb]: ditto

File: examples/optimizers_lazyadam.ipynb [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://github.com/tensorflow/addons/blob/master/tensorflow_addons/examples/optimizers_lazyadam.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
```
Comment (facaiy) [inline examples/optimizers_lazyadam.ipynb]: ditto

**Comment by facaiy** [comment]
@shashvatshahi1998 Welcome, Shashvat, thanks for the PR :-)

**Comment by WindQAQ** [comment]
@shashvatshahi1998 Don't forget to remove `tensorflow_addons/examples` dir. Btw, `git mv` would be useful enough in this case :-)

**Pull request closed without merging**